Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:7.10
```

Comment:
This is a not LTS version of Node. Should use 6 or 8. Node 8 seems to break some things in the react boilerplate for now. We are using Node 6.